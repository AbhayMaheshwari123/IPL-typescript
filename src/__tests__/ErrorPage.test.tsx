import renderer from 'react-test-renderer'
import Error from '../Components/ErrorPage'

it('Error',()=>{
    const component = renderer.create(<Error></Error>);
    const tree=component.toJSON();
    expect(tree).toMatchSnapshot();
})