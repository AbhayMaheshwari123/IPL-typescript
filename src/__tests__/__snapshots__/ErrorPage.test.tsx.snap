// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error 1`] = `
<div>
  <h1
    className="text-0-2-1"
  >
    Oops! Something Went Wrong
  </h1>
</div>
`;
